tile-group.lisp

```diff
@@ -1052,6 +1052,38 @@ space."
             (show-frame-indicator group))
           (run-hook-with-args *remove-split-hook* l frame)))))
 
+(defcommand (remove-sibling tile-group)
+    (&optional (group (current-group))
+	       (frame (let ((window-type (type-of (current-window))))
+			           (unless (eq window-type 'float-window)
+			             (window-frame (current-window))))))
+    ()
+  "removes the closest sibling frame to the specified frame, which defaults
+to the current frame. if the closes sibling is split, we call 
+remove-split-sibling, otherwise we just remove the sibling frame. "
+  (when frame
```

Comment:
Please make the doc string more descriptive of how to use this and what it is for. This is what will get compiled into the manual when its produced.  Include the rationale that you used to justify the pull request so that new users will understand what its for.

To add this to the manual, edit stumpwm.texi.in, find the appropriate place (I would put it near remove-split) and add this:
```
!!! remove-sibling
```